packages/phrases/src/locales/en.ts

```diff
@@ -156,6 +163,8 @@ const translation = {
       api_name: 'API name',
       api_name_placeholder: 'Enter your API name',
       api_identifier: 'API identifier',
+      api_identifier_tip:
+        'The unique identifier to the API resource. It must be an absolute URI and has no fragment component. Equals to the resource parameter in OAuth 2.0.',
```

Comment:
```suggestion
        'The unique identifier to the API resource. It must be an absolute URI and has no fragment (#) component. Equals to the resource parameter in OAuth 2.0.',
```

should align to the Chinese version?

```ts
      api_identifier_tip:
        '对于 API 资源的唯一标识符。它必须是一个绝对 URI 并没有 fragment (#) 组件。等价于 OAuth 2.0 中的 resource parameter。',
```